responses/configuration/set.go

```diff
@@ -8,26 +8,21 @@ import (
 
 //SetConfigResponse is for SMA to use when responding to an incoming request to PUT (i.e. UPDATE) a resource.
 type SetConfigResponse struct {
-	Code           string   `json:"code" yaml:"code,omitempty"`                     //HTTP code to return (e.g. 200, 500, etc.)
-	Description    string   `json:"description" yaml:"description,omitempty"`       //info allied with preceding HTTP code
-	ExpectedValues []string `json:"expectedValues" yaml:"expectedValues,omitempty"` //expected values to return
-	isValidated    bool     //internal member used for validation check
+	Success     bool   `json:"success"`               // HTTP success or failure
```

Comment:
Nitpick - not really HTTP success or failure; it's the success or failure of the set configuration request.